src/Service/CloudWatchLogs/tests/Unit/CloudWatchLogsClientTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace AsyncAws\CloudWatchLogs\Tests\Unit;
+
+use AsyncAws\CloudWatchLogs\CloudWatchLogsClient;
+use AsyncAws\CloudWatchLogs\Result\PutLogEventsResponse;
+use AsyncAws\CloudWatchLogs\ValueObject\InputLogEvent;
+use AsyncAws\CloudWatchLogs\ValueObject\PutLogEventsRequest;
```

Comment:
there's another issue with the generator here

this import should be AsyncAws\CloudWatchLogs\Input\PutLogEventsRequest instead

Reply:
Oh, that is true.. thank you